.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!main'
+
+env:
+  RUN: ${{ github.run_id }}-${{ github.run_number }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  PACKAGE_NAME: aws-iot-device-sdk-js
+  AWS_EC2_METADATA_DISABLED: true
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10.x]
```

Comment:
We should add 12 and 14 to this, shouldn't we?

Reply:
12 and 14 have build errors that look highly unpleasant to resolve (will require breaking change package upgrades and updating the code that gets broken).